lib/registries/addon/components/registration-form-view/styles.scss

```
@@ -19,3 +19,105 @@
 .Answer > p {
     font-size: 16px;
 }
+
+.NavigationMenu {
+    position: absolute;
+    z-index: 100;
+    top: 22px;
+    right: 30px;
+    width: 200px;
```

Instead of a static width, how about min/max widths? Forms with long titles look cramped.
```suggestion
    min-width: 200px;
    max-width: 300px;
```
(not sure if this would be necessary with `<ResponsiveDropdown>`)